README.md

```diff
@@ -47,24 +92,28 @@ can be done on the socket object outside of the connector)
 
 ## Performance
 
-Results achieved with `examples/speed_test_async_multi.cpp` for 1 thread, Intel Core i7-4800MQ, gentoo linux
+Results achieved with `examples/speed_test_async_multi.cpp` for 1 thread, Intel Core i7-8550U, void-linux, gcc 8.3.0
```

Comment:
is it necessary to measure and compare performance after compiling with clang too? And why gcc version 10 did not be used as mention in .travis.yml